mocks/mocks.go

```diff
@@ -289,13 +289,23 @@ func (sa *StorageAuthority) GetCertificate(_ context.Context, serial string) (co
 		return core.Certificate{
 			RegistrationID: 1,
 			DER:            certBlock.Bytes,
+			Issued:         sa.clk.Now().Add(-1 * time.Hour),
 		}, nil
 	} else if serial == "0000000000000000000000000000000000b2" {
 		certPemBytes, _ := ioutil.ReadFile("test/178.crt")
 		certBlock, _ := pem.Decode(certPemBytes)
 		return core.Certificate{
 			RegistrationID: 1,
 			DER:            certBlock.Bytes,
+			Issued:         sa.clk.Now().Add(-1 * time.Hour),
+		}, nil
+	} else if serial == "0000000000000000000000000000000000b3" {
+		certPemBytes, _ := ioutil.ReadFile("test/178.crt")
```

Comment:
How did you create 2 those original ones? What command do I need (Linux)

Reply:
I would probably try using `test/test-tools.go`'s newer `ThrowAwayCert` function for this. It will probably be the easiest way and I don't think you need a certificate with any particular properties.

Reply:
Should I activate it on a particular serial number?

Reply:
Hm, good point. Since ThrowAwayCert doesn't generate a certificate with a specific serial, we can't match it to the serial we're responding for. My recommendation is to add another helper function, ThrowAwayCertWithSerial, that takes as input a specific serial number and generates based on that. Then ThrowAwayCert should be redefined in terms of ThrowAwayCertWithSerial.

Reply:
Thank you, will implement

Reply:
@cpu done!